charts/nutanix-csi-storage/README.md

```diff
@@ -118,6 +118,7 @@ The following table lists the configurable parameters of the Nutanix-CSI chart a
 | `fileClass`                   | Activate Nutanix Files Storage Class                                                        | `false`                |
 | `fileClassName`               | Name of the Nutanix Files Storage Class                                                     | `nutanix-file`         |
 | `fileClassRetention`          | Retention policy for the Files Storage Class (Delete, Retain)                               | `Delete`               |
+| `dynFileSquashType`           | Squash type for Nutanix Dynamic File Storage (none, root-squash, all-squash)                | `root-squash`          |
```

Comment:
Please rename it to dynamicFileSquashType to keep it consistent with other parameters.

Reply:
Done